x/ibc/02-client/keeper/client.go

```diff
@@ -72,14 +71,13 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 		return nil, sdkerrors.Wrapf(err, "cannot update client with ID %s", clientID)
 	}
 
+	consensusHeight := clientState.GetLatestHeight()
```

Comment:
This should be `header.GetHeight()` since the update may be a past update to create a consensus state less than latest height